yb-voyager/src/issue/dml.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright (c) YugabyteDB, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package issue
+
+var advisoryLocksIssue = Issue{
+	Type:            ADVISORY_LOCKS,
+	TypeName:        "Advisory Locks",
+	TypeDescription: "",
+	Suggestion:      "",
+	GH:              "",
+	DocsLink:        "https://docs.yugabyte.com/preview/yugabyte-voyager/known-issues/postgresql/#advisory-locks-is-not-yet-implemented",
+}
+
+func NewAdvisoryLocksIssue(objectType string, objectName string, sqlStatement string) IssueInstance {
+	return newIssueInstance(advisoryLocksIssue, objectType, objectName, sqlStatement, map[string]interface{}{})
+}
+
+var systemColumnsIssue = Issue{
```

Comment:
why do we need a global variable for this? It can go inside `NewSystemColumnsIssue` constructor.